Jenkinsfile

```diff
@@ -31,7 +51,7 @@ node {
 
       govuk.setEnvar("RAILS_ENV", "test")
       govuk.bundleApp()
-      
+
       govuk.contentSchemaDependency()
```

Comment:
@suzannehamilton Ah! Right, that makes sense. Duly updated, both here and in alphagov/service-manual-publisher#379. Thanks for the review!